packages/fccanalyses/package.py

```diff
@@ -23,7 +23,7 @@ class Fccanalyses(CMakePackage, Key4hepPackage):
     depends_on('edm4hep')
     depends_on('py-awkward')
     depends_on('fcc-edm', when="@:0.2.9")
-    depends_on('acts@5.00.0', when="@0.3.0:")
+    depends_on('acts+dd4hep+tgeo+identification', when="@0.3.0:")
```

Comment:
This package (unlike k4actstracking) does not need `+dd4hep...`. The version should be updated though.

Reply:
moving to the latest acts seems to cause build failures.  Not sure what to do since keeping it as acts@5.00.0 rolls back to an earlier build